classes/04class/exercises/c04-rds01/luizfds/db-creation.txt

```diff
@@ -0,0 +1,121 @@
+aws ec2 create-security-group --group-name DevOpsAcademyEC2 --description "DevOpsAcademyEC2"
+{
+    "GroupId": "sg-07b6ea2ad1175ef17"
+}
+
+aws ec2 create-security-group --group-name DevOpsAcademyRDS --description "DevOpsAcademyRDS"
+{
+    "GroupId": "sg-0e9fbfe79714d1db5"
+}
+
+curl http://checkip.amazonaws.com/
+101.116.46.174
+
+# DevOpsAcademyEC2
+aws ec2 authorize-security-group-ingress \
+    --group-id sg-07b6ea2ad1175ef17 \
+    --protocol tcp \
+    --port 22 \
+    --cidr 101.116.46.174/24
+
+# DevOpsAcademyRDS
+aws ec2 authorize-security-group-ingress \
+    --group-id sg-0e9fbfe79714d1db5 \
+    --protocol tcp \
+    --port 3306 \
+    --cidr 101.116.46.174/24
```

Comment:
this one should be allowing ingress on 3306 from your subnet cidr.. not your home IP address (I'm assuming that 101.116.46.174) is your home IP.

Even if it's not and maybe is the instance public IP.. you should only allow the private ip's

in another note.. when referring only one IP you should use `/32` instead of `/24`

Reply:
Thanks Caio! I fixed that now, please review again.